sdk/python/feast/loaders/ingest.py

```diff
@@ -93,6 +96,10 @@ def _encode_pa_tables(file: str, fs: FeatureSet, row_group_idx: int) -> List[byt
     return feature_rows
 
 
+def normalize_fields(fields):
+    return {field.name: field.dtype for field in fields.values()}
```

Comment:
Should this be public? As @woop asked about why it's a method, if naming it does help readibility, you could nest the function in the one where it's used. Python supports that.